src/classes/SObjectFieldWrapper.cls

```diff
@@ -0,0 +1,29 @@
+public class SObjectFieldWrapper {
+    private final List<SObjectField> fields;
+
+    public SObjectFieldWrapper(SObjectField field) {
+        this(new List<SObjectField>{ field });
+    }
+
+    public SObjectFieldWrapper(List<SObjectField> fields) {
+        this.fields = fields;
+    }
+
+    public String Value {
```

Comment:
Lately, I've been leaning towards methods more than public properties lately - I think methods give more flexibility for extending/overriding/refactoring. I might change this to a method after it's merged, if that's cool with you.